modules/core/src/main/scala/fs2/kafka/ProducerSettings.scala

```diff
@@ -234,6 +234,7 @@ sealed abstract class ProducerSettings[F[_], K, V] {
 }
 
 object ProducerSettings {
+  val DefaultParallelism: Int = 10000
```

Comment:
I have a couple of concerns about this:
- I don't think it should be in `ProducerSettings` - it should just be a parameter to `pipe`.
- I'm not convinced it works as adverstised (controlling batch size). It limits the number of `ProduceResults` we're waiting for at any one time, but I'm not sure that's needed - the Java producer already maintains a buffer and blocks (with a default timeout of 60 seconds) when the buffer is full. Can't we just rely on that for backpressure and have unlimited parallelism on waiting for the results? (@vlovgr any thoughts on this?)

Reply:
I think we can have unbounded parallellism instead, yes.